source/georef/street_network.cpp

```diff
@@ -237,16 +237,27 @@ void PathFinder::start_distance_dijkstra(navitia::time_duration radius) {
 }
 
 std::vector<std::pair<type::idx_t, navitia::time_duration>>
+PathFinder::crow_fly_find_nearest_stop_points(navitia::time_duration radius, const proximitylist::ProximityList<type::idx_t>& pl) {
+    // Searching for all the elements that are less than radius meters awyway in crow fly
+    float crow_fly_dist = radius.total_seconds() * speed_factor * georef::default_speed[mode];
+
+    std::vector< std::pair<nt::idx_t, type::GeographicalCoord> > elements = pl.find_within(start_coord, crow_fly_dist);
+
+    std::vector<std::pair<type::idx_t, navitia::time_duration>> result;
+    for (auto element: elements) {
```

Comment:
not very important there, but remember, that this is doing a copy (of 3 int, so here it's not mandatory)